LICENSE.txt

```diff
@@ -1,27 +1,24 @@
-Copyright (c) 2014 Torchbox Ltd and individual contributors.
+Copyright (c) 2016, Divio AG
```

Comment:
2016?

Reply:
Yeah thats when there was the first commit :)